src/facade/dragonfly_connection.cc

```diff
@@ -769,6 +785,26 @@ auto Connection::IoLoop(util::FiberSocketBase* peer, SinkReplyBuilder* orig_buil
   do {
     FetchBuilderStats(stats_, orig_builder);
 
+    if (migration_request_ && cc_->subscriptions == 0) {
```

Comment:
let's add a comment why we can't migrate with subs: because it requires moving thread-local handles which is kinda annoying 😆